qiita_db/util.py

```diff
@@ -1506,13 +1509,15 @@ def get_artifacts_information(artifact_ids, only_biom=True):
                     main_query.root_id = pt.artifact_id)
             )
             SELECT * FROM has_target_subfragment
-            ORDER BY command_id, data_type, artifact_id
+            ORDER BY cid, data_type, artifact_id
             """
 
-        sql_params = """
-            SELECT parameter_set_name, array_agg(parameter_set) AS param_set
-            FROM qiita.default_parameter_set
-            GROUP BY parameter_set_name"""
+        sql_params = """SELECT default_parameter_set_id, command_id,
```

Comment:
I think in this query the param_set value will always have a single value. In the GROUP BY clause you've included the "default_parameter_set_id" which is the primary key of the table "default_parameter_set". Thus, it is impossible for two rows in the table to have the same default_parameter_set_id.

Reply:
right and in the code below we are only getting the first value [0], changing.